### DISCUSSION TITLE
Only winners or participants in epoch  2.1 will get free CML coupon in epoch 3. 0

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#38] [Ideas]
anyone who join the epoch 2.1 and at least plant a CML will be considered participants.
If they join the epoch  3.0, they will be issued CML coupon and COFFEE as initial funding. Others who did not join epoch 2.1 but registered to epoch 3.0 could be offered COFFEE initial fund but not (or very little) CML coupon

**Comment by geopor** [comment]
Contestants who already participated in the 2.1 epoch already have an advantage even without the extra CML seeds:
- they have learned mistakes / strategies that will help them in next round (like importance of keeping on top of CML seed loans).
- they have an easy signup path for the next epoch.

Additionally we're making it a priority to attract a larger group beyond the ~25 we had in first round. We don't want them to feel they're entering an unfair contest.

Instead of penalizing newcomers, I feel we should start everyone off at the same level as mentioned in https://github.com/tearust/teaproject/discussions/36

Since we're also interested to ensure all contestants have only 1 account, we can incentivize verifying with their socials by giving them CML coupons for:
1. Filling in at least one of their social accounts on the signup form.
2. They would also need to follow us on YouTube / Twitter / Medium / LinkedIn (depending on which account they filled out on the form). That step confirms they really own the account and also helps build our social accounts.

We discussed Telegram and although there is more concern with bots on Telegram, people who follow us there and participate in some discussion on our Telegram can also be verified. Some parts of the world may not have an easy path to register with the requested social accounts, so we can mention that Telegram is an option but that that the other social networks are preferred.

**Reply by kevingzhang** [reply]
See this issue https://github.com/tearust/tea-internal/issues/178  they are considered to be "investors" in the next epoch

**Reply by kevingzhang** [reply]
in the real life, there are differences between early stage investors and new miners.
Early stage investors owns initial seeds, but new miners have to buy from those investors.

Regard the KYC or real people verification is very hard, we have had a new idea as:
- Do not allow assets transfer (TEA, COFFEE or coupon)
- We will manually verify at the end of epoch, whoever violet this rule will be removed from the award qualification
- Everyone can join at any time, doesn't have to be in the google form at the beginning.

**Reply by kevingzhang** [reply]
We can actually enlarge our investor group. Besides those who already participate the previous epoch, anyone who regsitered in the google form can be investors too. this is the same as before. But whoever not in the google form but also want to join in the middle of epoch are new miners. 

This is a small problem for this: There will be too few new miners than investors. This is far from realistic

**Reply by kevingzhang** [reply]
BTW, based on https://github.com/tearust/tea-internal/issues/178 design, the CML seeds is no longer free. There will be a small debt. This is also the simulation of real life investors. They do pay USDT to TEAProject. they deserve the CML seeds.

Sorry for those who cannot access to our tea-internal repo, that is for TEA Project core team internal uses.

**Reply by geopor** [reply]
Ok I see, first round will be treated like investors.

> We can actually enlarge our investor group. Besides those who already participate the previous epoch, anyone who regsitered in the google form can be investors too. this is the same as before. But whoever not in the google form but also want to join in the middle of epoch are new miners.

Yes I think this is good if we allow the next group that onboard the possibility of signing up early to become investors. If they miss the deadline, they'll enter contest as miners.

**Comment by kevingzhang** [comment]
ok, we can do this way

**Comment by kevingzhang** [comment]
new idea.

The investors will not get free seeds. They just got a guaranteed price of seeds. 

The investors are those who registered before the epoch starts. So they will have coupons. 

Those coupons can redeem to frozen seeds at a **Guaranteed Price** but not free (as we did in epoch 2.1).

When they redeem the coupon, they will need to pay the price as 
A : 2000coffee
B: 1000 coffee
C: 500 coffee

Of course, the investors have the choice not to redeem them. 

Other non-investors, those who did not register and jump into the contest in the middle, won't have the coupon. They will have to buy seeds from the market at the market price. Very likely the investors will sell those seeds at the market at a higher price.

Where do they get initial funding? We offer unlimited COFFEE loans with a fixed interest rate in the next epoch. So anyone, including investors, can borrow COFFEE and pay for the seeds.

**Reply by geopor** [reply]
That will make it more fair. By requiring investors to pay for their initial seeds no one is getting a free lunch.

**Comment by kevingzhang** [comment]
We had a discussion yesterday with devs.

Because we currently redeem all CML coupons at once. It will require 6000 COFFEE if the investors have 1A 2B 4C. There is no choice for them to redeem only one seed. We do not want to change the code here due to the complicities. So it may be a big cost at the beginning. 

Can we give the investors a choice when they registering the contest. How many coupons they need.  They can choose 1A, 0B, and 1C if they want. So that they only need 2500 COFFEE to start.

Of course, we will have the cap. they cannot get more than the cap. 

If we choose this solution, we will modify the register google form, allow user to input number. @geopor

**Reply by geopor** [reply]
Ok sounds good, let me know when it is decided and I'll change the form.

For those already signed up to the form, we can assign them whatever the default selection is.

**Reply by kevingzhang** [reply]
You can wait few more days until the feature is tested. But you can update a test google form for our internal testing.

**Reply by geopor** [reply]
https://docs.google.com/forms/d/e/1FAIpQLSdFRVfpuNN6Gn5qhunRXDEuCO-R-ypDeSMzwZFYJ8s-BOB7iw/viewform?usp=sf_link

**Comment by kevingzhang** [comment]
a small enhancement. After the last question, could we add something to tell the user "what to do next", such as waiting for email, check spam email box. wait for announcement on discussion board, telegram etc.

**Reply by geopor** [reply]
Ok form has been updated
